buffalo/algo/base.py

```diff
@@ -37,13 +33,13 @@ def _normalize(self, feat):
         return feat
 
     def initialize(self):
-        self.__early_stopping = {'round': 0,
-                                 'min_loss': 987654321}
+        self.__early_stopping = {"round": 0,
+                                 "min_loss": 987654321}
```

Comment:
얘도 아래처럼 가도 되지 않을까요??

```suggestion
                                 "min_loss": float("inf")}
```